.gitignore

```diff
@@ -6,3 +6,5 @@
 pubspec.lock
 
 build/
+
+.idea/
```

Comment:
Can we add this to git ignore?

Reply:
yes we can. beacause not each developer uses intellij as IDE, this folder is generated only when you use intellij